packages/manager/modules/support/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "peerDependencies": {
     "@ovh-ux/manager-core": "^7.6.2",
+    "@ovh-ux/manager-models": "^0.0.0 || ^1.0.0",
```

Comment:
I'm not sure it is needed as a peer in fact, it's not an `angularjs` module

Reply:
damn i'm so used to work with angularjs modules that i forgot it wasn't one!